src/languages/en.js

```diff
@@ -149,6 +149,8 @@ export default {
     avatarCropModal: {
         title: 'Edit photo',
         description: 'Drag, zoom, and rotate your image to your preferred specifications',
+        rotateImage: 'Rotate',
+        zoomImage: 'Zoom',
```

Comment:
Since these two are just the generic words "rotate" and "zoom", can we move these to the common object? (`common.rotate` and `common.zoom`).

Reply:
Done